src/components/QuranReader/SidebarNavigation/SidebarSelectionSkeleton.tsx

```diff
@@ -0,0 +1,19 @@
+import styles from './SidebarSelectionSkeleton.module.scss';
+
+import Skeleton from 'src/components/dls/Skeleton/Skeleton';
+
+const ROWS_COUNT = 6;
```

Comment:
Salam brother @ahmedriad1 , jazakAllah for helping.

I think we'll need more than 6 row? For example here's how the sidebar height on my laptop, I think 6 will not cover the sidebar from top to bottom.
<img width="374" alt="image" src="https://user-images.githubusercontent.com/12745166/151082402-da5011a0-354a-401d-bd47-7902080c88a3.png">

What do you think?

Reply:
Salam bro @muhajirdev

Yeah should probably be. I used the settings drawer skeleton as a reference, and here is how it looked on larger resolutions: 
<img width="194" alt="image" src="https://user-images.githubusercontent.com/58295120/151109820-24ebd5ed-3d0b-4b7c-ac3e-e2e9efff81e2.png">

let me know what do you think, thanks

Reply:
I see, we probably should update the settings skeleton as well in the future.